tests/Utils/AstGetOperationAstTest.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+declare(strict_types=1);
+
+namespace GraphQL\Tests\Utils;
+
+use GraphQL\Language\Parser;
+use GraphQL\Utils\AST;
+use PHPUnit\Framework\TestCase;
+
```

Comment:
Would it make sense to add `@covers` annotation to allow this to influence `AST`'s coverage only?